AWSAuthSDK/Sources/AWSAuthUI/AWSAuthUIViewController.m

```diff
@@ -124,7 +125,16 @@ + (void)presentViewControllerWithNavigationController:(UINavigationController *)
 + (AWSAuthUIConfiguration *)getDefaultAuthUIConfiguration {
     
     AWSAuthUIConfiguration *authUIConfig = [[AWSAuthUIConfiguration alloc] init];
-    
+
+    if ([AWSAuthUIViewController isConfigurationKeyPresent:AWSInfoAppleIdentifier]) {
+        Class appleClass = NSClassFromString(@"AWSAppleSignInButton");
+        if (appleClass) {
+            [authUIConfig addAWSSignInButtonViewClass:appleClass];
+        } else {
+            AWSDDLogWarn(@"Found Apple sign in configuration in awsconfiguration.json but could not find dependencies. Skipping rendering in AuthUI");
```

Comment:
If you have to keep the `appleClass` check, this issue smells like an error, not a warn. If not, then you can remove it.

Reply:
Removed the warning